mem0/memory/storage.py

```diff
@@ -15,40 +15,43 @@ def _create_history_table(self):
                 CREATE TABLE IF NOT EXISTS history (
                     id TEXT PRIMARY KEY,
                     memory_id TEXT,
-                    prev_value TEXT,
+                    old_memory TEXT,
+                    new_memory TEXT,
                     new_value TEXT,
                     event TEXT,
-                    timestamp DATETIME,
+                    created_at DATETIME,
+                    updated_at DATETIME,
                     is_deleted INTEGER
                 )
             """
             )
```

Comment:
Can we please make this backward compatible? 

If table exists, the add columns and rename columns. Else, create new table from scratch.